tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,42 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Search Last Build
+    [Documentation]    Returns last build
+    [Arguments]    ${namespace}    ${word-to-search-build}
+    ${build} =    Run    oc get builds -n ${namespace} | grep ${word-to-search-build} | awk '{print $1;}'
+    [Return]    ${build}
+
+Delete Build
+    [Documentation]    Delete the build
```

Comment:
The style guide says it should be in the 3rd form (Deletes instead of Delete): 
_Deletes the build ${build_name} in  ${namespace}_